README.md

```diff
@@ -140,6 +155,8 @@ Some of the jobs we deploy require our AWS and [GCE](https://cloud.google.com/co
 * `aws_credentials` - AWS credentials in a form of shell include 
 * `gce_account` - google compute engine account.json file
 * `deployer_key:`- deployer key with elements: `name:` - name of the key and `key:` - the key itself
+* `secrets_py` - [Google Compute Engine](https://cloud.google.com/compute/) [credentials for Ansible](http://docs.ansible.com/guide_gce.html#calling-modules-with-secrets-py)
+* `gce_account_certificate_pem` - [Google Compute Engine](https://cloud.google.com/compute/)service account private key 
```

Comment:
Looks like a missing space in front of 'service' (the link and 'service' are next to each other without space).